docs/contribute/code.md

```diff
@@ -59,7 +59,7 @@ yarn build
 When asked if you want to start the build in `watch` mode, answer **yes** to develop in interactive mode. Afterward, choose which packages you want to build. For example, if you're going to work on a feature for `@storybook/addon-docs`, you might want to select `@storybook/addon-docs` and `@storybook/components`.
 
 <div class="aside">
-💡 Build's `watch' mode is great for interactive development. However, for performance reasons it only transpiles your code and doesn't execute the TypeScript compiler. If something isn't working as expected, try running `build` <b>WITHOUT</b> watch mode: it will re-generate TypeScript types and also perform type checking for you.
```

Comment:
Hey @huyenltnguyen I think there's no need to change everything to use `<code>`,

```
but rather just fix this from `watch' to `watch`
```

and likely all the formatting issues will go away. Can you please do that? thanks!